### PR TITLE
Lesson 4.2

### PR DESCRIPTION
**Pull request by SokolovskayaE**
Added Copyright text in Footer via JavaScript and Skills section.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+copyright.innerHTML = `&copy; Made with &hearts; by Elena Sokolovskaya ${thisYear}`;
+footer.appendChild(copyright);
+
+// Adding skills
+const skills = ["JavaScript", "HTML", "CSS", "GIT"];
+const skillsSection = document.getElementById("Skills");
+const skillsList = skillsSection.querySelector("ul");
+for (let i = 0; i < skills.length; i++) {
+  const skill = document.createElement("li");
+  skill.innerHTML = `${skills[i]}`;
```
Comment (toma63) [inline js/index.js]: Could just use skills[i] since it is a string already.

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const footer = document.querySelector("footer");
+const copyright = document.createElement("p");
+copyright.innerHTML = `&copy; Made with &hearts; by Elena Sokolovskaya ${thisYear}`;
+footer.appendChild(copyright);
+
+// Adding skills
+const skills = ["JavaScript", "HTML", "CSS", "GIT"];
+const skillsSection = document.getElementById("Skills");
+const skillsList = skillsSection.querySelector("ul");
+for (let i = 0; i < skills.length; i++) {
```
Comment (toma63) [inline js/index.js:13]: Slightly simpler to use for (let skill of skills) { ... }

**Review by toma63 — APPROVED** [review]
Looks good.  A couple of minor suggestions in comments.

File: js/index.js (hunk 1 above)
Comment (SokolovskayaE) [reply js/index.js]: Thank you! I just fixed it.

File: js/index.js (hunk 2 above)
Comment (SokolovskayaE) [reply js/index.js:13]: I didn't fully understand for ... of loop, so I decided to write as I understand it at this stage.

**Pull request merged**